app/src/main/java/org/eyeseetea/malariacare/domain/entity/Configuration.java

```diff
@@ -16,75 +18,64 @@
 
         public CountryVersion(String uid, String country, int version, Date lastUpdate,
                 String reference, String name) {
-            this.uid = uid;
-            this.country = country;
-            this.version = version;
-            this.lastUpdate = lastUpdate;
-            this.reference = reference;
-            this.name = name;
-        }
 
-        public CountryVersion(){}
-
-        private CountryVersion(Builder builder) {
-            setUid(builder.uid);
-            setCountry(builder.country);
-            setVersion(builder.version);
-            setLastUpdate(builder.lastUpdate);
-            setReference(builder.reference);
-            setName(builder.name);
+            setUid(required(uid,"UID is required"));
```

Comment:
Can you remove private setters? It's not necessary we have private setters, so we reduce the size of the class

Reply:
Sure!